<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paaikkuna</class>
 <widget class="QMainWindow" name="Paaikkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paaikkuna</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSpinBox" name="mittari_km_h">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> km/h</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button_nollaa">
      <property name="text">
       <string>Nollaa</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_mailia_h">
      <property name="text">
       <string>mailia/h</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QScrollBar" name="liukuri_km_h">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="button_lopeta">
      <property name="text">
       <string>Lopeta</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLCDNumber" name="mittari_mailia_h">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKm_h_mailia_h">
    <property name="title">
     <string>km/h -&gt; mailia/h</string>
    </property>
   </widget>
   <addaction name="menuKm_h_mailia_h"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>button_lopeta</sender>
   <signal>clicked()</signal>
   <receiver>Paaikkuna</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mittari_km_h</sender>
   <signal>valueChanged(int)</signal>
   <receiver>liukuri_km_h</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>liukuri_km_h</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mittari_km_h</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
